include/deal.II/base/tensor.h

```diff
@@ -1515,6 +1515,30 @@ operator * (const Tensor<rank_1, dim, Number> &src1,
 }
 
 
+/**
+ * The outer product of two tensors of @p rank_1 and @p rank_2: Returns a
+ * tensor of rank $(\text{rank\_1} + \text{rank\_2})$:
+ * @f[
+ *   \text{result}_{i_1,..,i_{r1},j_1,..,j_{r2}}
+ *   = \text{left}_{i_1,..,i_{r1}} \cdot \text{right}_{k, j_1,..,j_{r2}.}
```

Comment:
no `\cdot` here, it's just a regular product between scalars.

also, no index `k` on the right.